src/test/ReactTestUtils.js

```diff
@@ -12,23 +12,27 @@
 'use strict';
 
```

Comment:
While we're at it can we move `src/test` to a more appropriate place (considering the package structure)?